### PR TITLE
Add Vercel to Deno hosting solutions

### PR DESCRIPTION
**Pull request by kidonng**

### CODE REVIEW

File: site/docs/resources/faq.md [hunk 1]
```diff
@@ -54,3 +54,4 @@ Here's are some options to where you can host your Deno app:
 1. [Cloudflare Workers](https://workers.dev)
 2. [Deno Deploy](https://deno.com/deploy)
 3. Heroku, [read here](https://dev.to/ms314006/deploy-your-deno-apps-to-heroku-375h)
```
Comment (KnorpelSenf) [inline site/docs/resources/faq.md:56]: @WingLim please translate “read here” in the Chinese version.

**Review by KnorpelSenf — APPROVED** [review]
Thanks!

File: site/docs/resources/faq.md (hunk 1 above)
Comment (kidonng) [reply site/docs/resources/faq.md:56]: I roughly translated it into "参考此文" (refer to this article), does it look good to you?

Comment (WingLim) [reply site/docs/resources/faq.md:56]: LGTM, great job👏

**Pull request merged**